cmd/minikube/cmd/ssh.go

```diff
@@ -28,7 +28,7 @@ import (
 
 // sshCmd represents the docker-ssh command
 var sshCmd = &cobra.Command{
-	Use:   "ssh",
+	Use:   "ssh ['COMMAND' or -- COMMAND]",
```

Comment:
I don't think this is quite right. Single and double quotes make a difference and depending what you want are valid as well. Using single quotes will not interpolate anything into the command which might or might not be what you want - http://www.howtogeek.com/howto/29980/whats-the-difference-between-single-and-double-quotes-in-the-bash-shell. It will also depend on the shell type. I think the best we can do it generally use [command].